libcloud/compute/drivers/ec2.py

```diff
@@ -1317,87 +1497,6 @@
             'os1.8xlarge'
         ]
     },
-    'eu-west-2': {
```

Comment:
@tonybaloney No it hasn't been removed conmpletely, only the duplicate entries: it's a JSON dictionary, where the key is unique and only the last(?) entry wins:
Warning: the following script will create the files [1-8].py in your current directory!
```
curl -s https://raw.githubusercontent.com/apache/libcloud/trunk/libcloud/compute/drivers/ec2.py | sed -ne '
1567,1593w 1.py
1594,1620w 2.py
1621,1647w 3.py
1648,1674w 4.py

1729,1755w 5.py
1756,1782w 6.py
1783,1809w 7.py
1810,1836w 8.py
'
diff 1.py 2.py ; diff 2.py 3.py ; diff 3.py 4.py
diff 5.py 6.py ; diff 6.py 7.py ; diff 7.py 8.py
```